Getting-Started.md

```diff
@@ -1,55 +1,54 @@
+# Setting up your environment
```

Comment:
This file has been removed as the tutorial has been published on IBM developer

Reply:
There might be some legacy links to these files out there, can we instead replace the superseded markdown files with links to their new location on IBM Developer?

Reply:
For example, that's what this file does in the Speech repo: https://github.com/ibm-build-lab/Watson-Speech/blob/main/Speech%20To%20%20Text/Speech-To-Text-Tutorial.md

Reply:
Merged my branch with your updates to main to include these changes.